bin/get_github_release.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+"""Download the latest version of github-release and place in a temporary location"""
+
+from __future__ import unicode_literals, print_function
+
+import platform
+import subprocess
+import tempfile
+
+import os
+import re
+import requests
+
+URL = "https://api.github.com/repos/aktau/github-release/releases/latest"
+ASSET_NAME_PATTERN = re.compile(r"^{}-amd64-github-release\.tar\.bz2$".format(platform.system().lower()))
+
+
+def main():
+    release_info = _get_release_info()
+    asset_url = _find_wanted_asset(release_info)
+    tmp_dir, asset_path = _download_asset(asset_url)
+    bin_path = _unpack(tmp_dir, asset_path)
+    print(bin_path)
+
+
+def _unpack(tmp_dir, asset_path):
+    old_dir = os.getcwd()
+    os.chdir(tmp_dir)
```

Comment:
Could we use the `tar --directory=tmp_dir` to avoid having to chdir ?

Reply:
Sounds like an excellent suggestion.